.github/workflows/install-kotlin-deps/action.yml

```diff
@@ -0,0 +1,31 @@
+# This action downloads the Kotlin dependencies directly from Maven
+# Central. We should to be able to use Maven for this, and the
+# following works fine on Linux and macOS:
+#
+# mvn --no-transfer-progress dependency:copy \
+#   "-Dartifact=net.java.dev.jna:jna:5.14.0" \
+#   -Dmdep.stripVersion=true -DoutputDirectory=$PWD
+#
+# However, it fails on Windows, saying
```

Comment:
Despite my best attempts, the Windows tests now fail with a much simpler error:

```text
   Compiling mls-rs-uniffi v0.1.0 (D:\a\mls-rs\mls-rs\mls-rs-uniffi)
    Finished dev [unoptimized + debuginfo] target(s) in 26.04s
test simple_scenario_sync ... FAILED

failures:

---- simple_scenario_sync stdout ----
Creating testing out_dir: D:\a\mls-rs\mls-rs\target\tmp\mls-rs-uniffi-3416e5e0eb569dae
Error: Failed to spawn `kotlinc` to compile the bindings

Caused by:
    program not found


failures:
    simple_scenario_sync

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 43.62s
```

According to [Windows2022-Readme.md](https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md), Kotlin 1.9.23 should be installed. I can also see it working:

```
$ which kotlinc
/c/tools/kotlinc/bin/kotlinc
$ kotlinc -version
info: kotlinc-jvm 1.9.23 (JRE 1.8.0_402-b06)
```

The UniFFI test somehow doesn't see it — I looked briefly at [test.rs](https://github.com/mozilla/uniffi-rs/blob/main/uniffi_bindgen/src/bindings/kotlin/test.rs), but I don't see it overriding the `$PATH` anywhere.

I would suggest disabling the tests on Windows for now. What do people here think?

Reply:
I'm fine with disabling windows Kotlin

Reply:
Thanks, I'll refactor the PR to do that!